.github/workflows/spell-checker.yml

```diff
@@ -1,25 +1,33 @@
-name: Fix typos nightly
+name: Spell Check
 
 on:
-  workflow_dispatch: {} # enable manual running of workflow.
+  pull_request:
```

Comment:
Do we need to run on every PR? any thoughts @gjermundgaraba @DimitrisJim ?

Reply:
fine with w/e, if it gets in people's way can be tweaked in future. (if sdk folks haven't rugged it, it might not be that bad)